### PR TITLE
fix(airbyte-ci): cleanup unused / empty schema directories after inlining schemas to manifests

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

This pull request does two things:
1. `connectors migrate-to-inline_schemas` will remove unused remaining json schemas and schemas directory.
2. formatting helper uses logger for output so running this in CLI looks clean and nice.

### CODE REVIEW

File: airbyte-ci/connectors/pipelines/README.md [hunk 1]
```diff
@@ -773,7 +773,8 @@ E.G.: running Poe tasks on the modified internal packages of the current branch:
 
 | Version | PR                                                         | Description                                                                                                                  |
 | ------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| 4.28.1  | [#42972](https://github.com/airbytehq/airbyte/pull/42972)  | Add airbyte-enterprise support for format command.                                                                           |
+| 4.28.2  | [#43210](https://github.com/airbytehq/airbyte/pull/43210)  | `migrate-to-inline_schemas` removes unused schema files and empty schema dirs.                                               |
```
Comment (natikgadzhi) [inline airbyte-ci/connectors/pipelines/README.md:776]:
```suggestion
| 4.28.2  | [#43297](https://github.com/airbytehq/airbyte/pull/43297)  | `migrate-to-inline_schemas` removes unused schema files and empty schema dirs.                                               |
```

**Comment by natikgadzhi** [comment]
Hmm, that's weird.

@alafanechere could it somehow be connected to your work on always coupling streict-encrypt connectors? 

```
___________ test_get_selected_connectors_with_modified_and_language ____________

    def test_get_selected_connectors_with_modified_and_language():
        first_modified_connector = pick_a_random_connector(language=ConnectorLanguage.PYTHON)
        second_modified_connector = pick_a_random_connector(language=ConnectorLanguage.JAVA, other_picked_connectors=[first_modified_connector])
        modified_files = ***first_modified_connector.code_directory / "setup.py", second_modified_connector.code_directory / "setup.py"***
        selected_connectors = connectors_commands.get_selected_connectors_with_modified_files(
            selected_names=(),
            selected_support_levels=(),
            selected_languages=(ConnectorLanguage.JAVA,),
            modified=True,
            metadata_changes_only=False,
            metadata_query=None,
            modified_files=modified_files,
        )
    
>       assert len(selected_connectors) == 1
E       AssertionError: assert 2 == 1
E        +  where 2 = len([ConnectorWithModifiedFiles(relative_connector_path='source-clickhouse', modified_files=frozenset(***PosixPath('airbyte-...)), ConnectorWithModifiedFiles(relative_connector_path='source-clickhouse-strict-encrypt', modified_files=frozenset())])
```

**Review by girarda — APPROVED** [review]
:shipit:

**Pull request merged**